design/BulkAPI.md

```diff
@@ -309,4 +309,67 @@ kruizeconfigjson: |
 ## Bulk API Threads
 
 - **Control Mechanism:** The number of threads used for bulk API operations can be controlled using the environment
-  variable `bulkThreadPoolSize`.
\ No newline at end of file
+  variable `bulkThreadPoolSize`.
+
+## Experiment Name Format Configuration
+
+- **experimentNameFormat:** The `experimentNameFormat` environment variable is used to define the format for experiment
+  names. For example, if the
+  experiment name should follow the structure:
+
+```
+org_id|source_id|cluster_id|namespace|k8s_object_type|k8s_object_name
+```
+
+then set or define the `experimentNameFormat` as follows:
+
+```
+"experimentNameFormat": "%label:org_id%|%label:source_id%|%label:cluster_id%|%namespace%|%workloadtype%|%workloadname%|%containername%"
```

Comment:
Should we remove %containername% from here?

Reply:
its optional

Reply:
Yes ROS might remove it